pep-0595.rst

```diff
@@ -370,7 +370,7 @@ need to be addressed regardless of the approach used:
 
   Since Roundup converts references to links when messages are
   requested, it is possible to update the target and generate the
-  correct link.  This need already arose several times, for
+  correct link.  This need already arised several times, for
```

Comment:
Actually, "arose" is correct here. So it seems there's nothing to do in this PR?